repos/fdbt-site/src/interfaces/typeGuards.ts

```diff
@@ -178,3 +179,11 @@ export const isMultiOperatorMultipleServicesTicket = (
     (ticket as MultiOperatorMultipleServicesTicket).additionalOperators &&
     (ticket as MultiOperatorMultipleServicesTicket).additionalOperators.length > 0 &&
     'selectedServices' in ticket;
+
+export const isFlatFarePricedByDistanceTicket = (
+    ticket: Ticket | WithIds<Ticket>,
+): ticket is FlatFareMultipleServices =>
+    !!ticket &&
```

Comment:
I dont think we need to do !!ticket because the function take Ticket | WithIds<Ticket> as arguments, not undefined.